demosplan/DemosPlanCoreBundle/Permissions/Permissions.php

```diff
@@ -279,6 +279,7 @@ protected function setGlobalPermissions(): void
                 'feature_json_api_update',
                 'feature_orga_edit_all_fields',
                 'feature_orga_get',
+                'feature_organisation_user_list',
```

Comment:
may be we can comment this too something like 'im Gegensatz zur Berechtigung „area_organisations“ ermöglicht die Berechtigung „feature_organisation_user_list“ die Anzeige der Organisationsliste, gewährt aber keinen Zugriff auf das Element „Organisation“ im Menü'

Reply:
done in [1abaffb](https://github.com/demos-europe/demosplan-core/pull/3475/commits/1abaffb4f36a33fd142e6a0a6e1acaeb8395060f)